extensions/photoroom-image-editing/README.md

```diff
@@ -0,0 +1,3 @@
+# Photoroom Image Editing
+
+Resize images, remove and change the background, add shadows and much more!
```

Comment:
I would rather a longer README, listing all the capabilities. "Much more" doesn't say much.
Also, explaining that you can submit multiple images for instance. It was not clear to me at the beginning.

Reply:
Makes sense, done! ✅